README.md

```diff
@@ -0,0 +1,14 @@
+#Hello Microverse Project
+
+##Built With
+
+- HTML
+- CSS
+
+##Getting Started
+This project's about how to create a project on GitHub and set up a linter for HTML&CSS, and make a new branch to implement the "hello microverse" functionality.
+
+##Authors
+###Luis Muñoz
+
+- GitHub: @lmunozm1702
```

Comment:
- [ ] Please add links to your GitHub profile so that people can visit your profile by clicking on the link.

Reply:
It's done.